bin/plugin/commands/get-plugin-dir.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+/**
+ * Internal dependencies
+ */
+const { log } = require( '../lib/logger' );
+
+exports.options = [
+	{
+		argname: '-s, --slug <slug>',
+		description: 'Slug to search out whether it is a plugin or module.',
+	},
+];
+
+/**
+ * Command to get directory for plugin/module based on the slug.
+ *
+ * @param {Object} opt Command options.
+ */
+exports.handler = async ( opt ) => {
+	doRunGetPluginDir( {
+		pluginsJsonFile: 'plugins.json', // Path to plugins.json file.
+		slug: opt.slug, // Plugin/module slug.
+	} );
+};
+
+/**
+ * Prints directory root for plugin or module based on the slug.
+ *
+ * @param {Object} settings Plugin settings.
+ */
+function doRunGetPluginDir( settings ) {
+	if ( settings.slug === undefined ) {
+		throw Error( 'A slug must be provided via the --slug (-s) argument.' );
+	}
+
+	// Resolve the absolute path to the plugins.json file.
+	const pluginsFile = path.join(
+		__dirname,
+		'../../../' + settings.pluginsJsonFile
+	);
+
+	try {
+		// Read the plugins.json file synchronously.
+		const { modules, plugins } = require( pluginsFile );
```

Comment:
Question: In `test-plugns.js` it is doing:

```json
const pluginsConfig = JSON.parse( pluginsJsonFileContent );
```

Shouldn't it rather use `require` like is used here? At least the same loading approach should be used between the two files.

Reply:
Updated in https://github.com/WordPress/performance/pull/1000/commits/15b1cc0404f04e5e491ec36f0f5c3e7b764f2a86